spec/ace.md

```diff
@@ -49,6 +52,22 @@ Volumes that are specified in the Pod Manifest are mounted into each of the apps
 For example, say that the worker-backup and reduce-worker both have a `mountPoint` named "work".
 In this case, the executor will bind mount the host's `/opt/tenant1/work` directory into the `path` of each of the matching "work" `mountPoint`s of the two app filesystems.
 
+If the target `path` does not exist in the rendered filesystem, it SHOULD be created, including any missing parent directories.
+If the target `path` is a non-empty directory, its contents SHOULD be discarded (e.g. obscured by the bind mount).
+If the target `path` refers to a file, the ACE SHOULD remove that file and create a directory in its place.
```

Comment:
The behavior described in lines 56 and 57 sound like error-hiding to me. That's ok...but perhaps it should be noted that the ACE _SHOULD_ provide a warning that existing files are being masked by a volume. An empty directory target would however be expected.